setup.py

```diff
@@ -23,7 +23,7 @@ def read(*parts):
 
 install_requires = [
     'Django>=2.2',
-    'celery>=3.0,<5.0',
+    'celery>=4.0',
```

Comment:
Have you tested it with Celery 5.x ? As mentioned in the issue, we haven't, I don't want to change this unless I know for sure it works.